Obedient-Lobsters/WorkforceManagement
# Department: Creation of index method, view file Index, styling changes to Layout.

# Description
Creation of index method and view for Department. Styling changes to Layout. Update to README will be addressed in a separate pull request
 Fixes #5
 ## Type of change
 # Testing Instructions
Start SQL Server Management Studio 

Run BangazonDatabaseCreateSeed.sql in SSMS

Update the appsetting.json in Visual Studio community to reflect your desktop and your server

Start the program by cd'ing into the WorkforceManagement (make sure you are deep enough, the .sln file is a few WorkforceManagement directories deep) and using the command `dotnet run`.

Once the program is running, navigate to your browser and go to the localhost address shown in the command prompt that opens up when you run the program. Click on the **Departments** tab in the navbar, and you should see a list with all Departments in the database.
